service/controller/app/v1/resource/chart/create_test.go

```diff
@@ -6,26 +6,15 @@ import (
 	"testing"
 
 	"github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
-	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
 	"github.com/giantswarm/apiextensions/pkg/clientset/versioned/fake"
-	"github.com/giantswarm/micrologger"
 	"github.com/giantswarm/micrologger/microloggertest"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/client-go/kubernetes"
 	k8sfake "k8s.io/client-go/kubernetes/fake"
 
 	"github.com/giantswarm/app-operator/kubeconfigtest"
```

Comment:
As discussed this package should be `github.com/giantswarm/kubeconfig/kubeconfigtest`. Let's get this fixed having it in the wrong repo is messy.

Reply:
I'm not sure about moving it to `kubeconfig` since then it will use app-operator repo as well, 
https://github.com/giantswarm/app-operator/blob/93d450bfd4aaccfc2df80d06b9c52ec76b4c8824/kubeconfigtest/kubeconfig.go#L9

should we move this `kubeconfig.Config` struct too?

Reply:
Ack. This is a problem with the kubeconfig service I introduced :(

The problem is this file is in the wrong place and needs to be removed.

https://github.com/giantswarm/app-operator/blob/master/kubeconfigtest/kubeconfig.go

Let's just use the current kubeconfig service with fake clientsets. 

```
			var kc kubeconfig.KubeConfig
			{
				c := kubeconfig.Config{
					G8sClient: fake.NewSimpleClientset(),
					K8sClient: k8sfake.NewSimpleClientset(),
					Logger:    microloggertest.New(),
				}

				kc, err := kubeconfig.New(nil)
				if err != nil {
					t.Fatalf("error == %#v, want nil", err)
				}
			}

			c := Config{
				G8sClient:      fake.NewSimpleClientset(),
				K8sClient:      k8sfake.NewSimpleClientset(),
				KubeConfig:     kc,
				Logger:         microloggertest.New(),
				WatchNamespace: "default",
			}
			r, err := New(c)
			if err != nil {
				t.Fatalf("error == %#v, want nil", err)
			}
```

It makes the test longer but its the best option for now. Next week I'll take another look at the kubeconfig service and see if this can be improved.